src/libs/Notification/PushNotification/index.native.ts

```diff
@@ -148,45 +147,41 @@ function deregister() {
  *       if we attempt to bind two callbacks to the PushReceived event for reportComment notifications,
  *       the second will overwrite the first.
  *
- * @param {String} notificationType
- * @param {Function} callback
- * @param {String} [triggerEvent] - The event that should trigger this callback. Should be one of UrbanAirship.EventType
+ * @param triggerEvent - The event that should trigger this callback. Should be one of UrbanAirship.EventType
  */
-function bind(notificationType, callback, triggerEvent) {
-    if (!notificationEventActionMap[triggerEvent]) {
-        notificationEventActionMap[triggerEvent] = {};
+function bind(notificationType: string, callback: NotificationEventActionCallback, triggerEvent: EventType) {
```

Comment:
Proof that this change works [here](https://www.typescriptlang.org/play?ts=5.2.2#code/C4TwDgpgBAcg9sAlgM0QYwIZLgOwKIBuEOwAgmtjgMIYA2tARhmgNZQC8UAFACZYYAuKBhwgAlBwB8UAnEQ8A3AFgAUKrS4AzsCg4EKdFkS5CxMhWM4AshjBCAChgBOSOgB4AShA1Oeb7U6IOADmADRQXj5+AUFhsPqomJSmJOSUNPRMrJI5HFAA3gC+ymoqyACuOBa4UAxBPFx6SIlGuAAq4BBCMSHhmJnMLELwzYbJRKnV1HSMg+HAgcHBEE4pwN0LsRL5qlBQtBA6zJQ2YHlNBkmWa2mWpwDam0srawC6JbtQKNwAhMd3tm2nz2e3+uFOeSKJT2hVUnzB1ls9wuLUoHUgrzy-VmrGhugSY2uE3MJyRT2Wq2JmM4CNOJVhpQ0OE0cAOADpaHBglwAOQowkmYm3cG2HnhABSAGUAPIwNk9YIoECNAlXQVmYWIsBiMQlOo4Bp8gCMYqgRvCPIgJt16i0rIgHK5vP5avwQqmp1NUtl8s2ISVKtGrpuHsBNpU+sNOAATKboxarTzw0yWezOdy+arWm6NaGwF6ZXKFQGXdmQ6TteHI86TeEAMwJ2PJu1pp2ZoNl90Vgs+4vIZWl8a5is6vX1Z2x8IAFgT1pKKftjozg6Jw4B+Ylhd9gX9-cDl07a5FlZKQA).